src/main/java/org/jabref/gui/externalfiletype/ExternalFileType.java

```diff
@@ -1,15 +1,21 @@
 package org.jabref.gui.externalfiletype;
 
+import javafx.beans.property.StringProperty;
+
 import org.jabref.gui.icon.JabRefIcon;
 import org.jabref.model.entry.field.Field;
 import org.jabref.model.entry.field.FieldFactory;
 
 public interface ExternalFileType {
-    String getName();
+    StringProperty getName();
+
+    String getNameAsString();
 
-    String getExtension();
+    StringProperty getExtension();
```

Comment:
Where is the `StringProperty` used? I always saw `getValue()`. Did I miss something?